sapl/base/views.py

```diff
@@ -1987,6 +1987,46 @@ def get(self, request, *args, **kwargs):
         def post(self, request, *args, **kwargs):
             return self.get(request, *args, **kwargs)
 
+    class UpdateView(CrudAux.UpdateView):
+        
+        form_class = ConfiguracoesAppForm
+
+        def form_valid(self, form):
+            numeracao = AppConfig.objects.last().sequencia_numeracao_protocolo
```

Comment:
A lógica não é bem essa não... (infelizmente!) O que devemos preservar é a forma que estava. Se existir um número inicial e ele for maior que o número do ano existente usa ele **na hora de criar o protocolo** senão usa a regra que tiver sido configurada. O que vai mudar é que agora ao invés de estar na tela de cadastro de protocolo, esse campo vai estar em tabelas auxiliares.

<img width="563" alt="Screen Shot 2020-05-25 at 16 01 09" src="https://user-images.githubusercontent.com/9326037/82837986-18219a00-9ea1-11ea-8fe5-b29e4817d1af.png">

Reply:
Lógica alterada.